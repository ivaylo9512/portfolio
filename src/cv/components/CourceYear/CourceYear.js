const CourceYear = ({ years, icon }) => {
    return(
        <div>
            {years}
            {icon}
        </div>
    )
}

export default CourceYear