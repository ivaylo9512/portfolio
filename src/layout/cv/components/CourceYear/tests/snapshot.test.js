import CourceYear from '../CourceYear';
import { shallow } from 'enzyme';

describe('CourceYear snapshot tests', () => {
    const createWrapper = () => shallow(
        <CourceYear />
    )

    it('should match snapshot', () => {
        const wrapper = createWrapper();
     
        expect(wrapper).toMatchSnapshot();
    })
});