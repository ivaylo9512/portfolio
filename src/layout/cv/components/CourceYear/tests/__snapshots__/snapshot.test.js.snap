// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CourceYear snapshot tests should match snapshot 1`] = `<div />`;
